Source/CronosPlayUnreal/Private/DefiWalletCoreActor.cpp

```diff
@@ -52,7 +55,14 @@ ADefiWalletCoreActor::ADefiWalletCoreActor()
 
   _coreWallet = NULL;
 
-  UPlayCppSdkBPLibrary::SetupUserAgent("CronosPlay-UnrealEngine-Agent");
+  IPluginManager &PluginManager = IPluginManager::Get();
+  TSharedPtr<IPlugin> Plugin = PluginManager.FindPlugin("CronosPlayUnreal");
```

Comment:
if this fails, will it throw an exception?

Reply:
This return Pointer to the plugin's information, or nullptr. It is inside the Plugin's code, it finds itself. The code will work only if the user enable and use the plugin, so it would not fail.